index.html

```diff
@@ -0,0 +1,105 @@
+<!-- Declare document type as HTML -->
+<!DOCTYPE html>
+<!-- Start HTML document -->
+<html lang="en">
+  <head>
+    <!-- Declare character set -->
+    <meta charset="UTF-8" />
+    <!-- Declare compatibility with Internet Explorer -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <!-- Declare viewport configuration -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Set title of the webpage -->
+    <title>My Personal Portfolio</title>
+    <!-- Link to external stylesheet -->
+    <link rel="stylesheet" href="css/styles.css" />
+    <!-- Include Fontawesome icons kit -->
+    <script
+      src="https://kit.fontawesome.com/7f35089b8a.js"
+      crossorigin="anonymous"
+    ></script>
+    <!-- Include Google Fonts link -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- Start header section -->
+    <header>
+      <!-- Start container -->
+      <div class="container">
+        <!-- Start logo section -->
+        <div class="logo-wrapper">
+          <a href="#" class="logo">My Logo</a>
+        </div>
+        <!-- End logo section -->
+        <!-- Start menu section -->
+        <div class="menu-bars" id="menu-toggle">
+          <i class="fa-solid fa-regular fa-bars"></i>
+        </div>
```

Comment:
- [ ]  For semantics reasonsthe hamburger button should have a nav tag.

Reply:
Hi @algerina, I didn't understand what you're emphasizing here. Please explain to me more so I don't lose another review chance. Just let me know where exactly I need to put my nav tag, either inside the "menu-bars" div or inside the (i) tag. Truely appreciate your time and help. Moreover, at this point we are not supposed to add any javascript code so I guess only nav tag is require here if I'm not wrong.